core/test/nuxeo-operation.test.js

```diff
@@ -0,0 +1,179 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { fixture, html, waitChanged } from './test-helpers';
+import '../nuxeo-operation.js';
+import '../nuxeo-page-provider.js';
+
+/* eslint-disable no-unused-expressions */
+suite('<nuxeo-operation>', () => {
+  const responseHeaders = {
+    json: { 'Content-Type': 'application/json' },
+    plain: { 'Content-Type': 'text/plain' },
+  };
+
+  let server;
+  setup(() => {
+    server = sinon.fakeServer.create();
+    server.autoRespond = true;
+  });
+
+  teardown(() => {
+    server.restore();
+  });
+
+  suite('when executing an operation', () => {
+    setup(() => {
+      server.respondWith('GET', '/json/cmis', [200, responseHeaders.json, '{}']);
+      server.respondWith('POST', '/api/v1/automation/login', [
+        200,
+        responseHeaders.json,
+        '{"entity-type":"login","username":"Administrator"}',
+      ]);
+      server.respondWith('GET', '/api/v1/user/Administrator', [
+        200,
+        responseHeaders.json,
+        '{"entity-type":"user","username":"Administrator"}',
+      ]);
+    });
+
+    test('should retrieve something valid', async () => {
```

Comment:
I know that it is not the scope of this ticket, but I'm not sure if I understand the purpose of this specific test...

Reply:
it's the basic "Hey, it works" use case, meaning it does not throw any exception. I did not really review the existing test tbh though, just migrated things over...